client/state/selectors/get-locale-suggestions.js

```diff
@@ -0,0 +1,15 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns an object of localized language names
+ *
+ * @param  {Object}  state Global state tree
+ * @returns {Array|Null} an array of guessed locales for the user
+ */
+export default function getLocaleSuggestions( state ) {
+	return get( state.i18n.localeSuggestions, 'items', null );
```

Comment:
If we bother using get, I normally like bulletproofing the whole thing. eg `get( state, 'i18n.localeSuggestions.items', null )`